### ISSUE TITLE
'Access is denied' when installing CPU-Z

### ISSUE THREAD

**Issue by nitishvijai** [microsoft/winget-pkgs#2428]
Create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new?assignees=&labels=&template=Bug_Report.md&title=}

# Brief description of your issue
Whenever I try to install CPU-Z using winget, the exe has successfully downloaded and hash verified, but I get an 'Access is denied' error.

# Steps to reproduce

1. Open Command Prompt or PowerShell in Windows Terminal.
2. Type "winget install CPU-Z"

# Expected behavior

CPU-Z will successfully install.

# Actual behavior

The installer returns an 'Access is denied' error, and the terminal reports "Installer failed with exit code: 1".

# Environment

```none
[winget --info]
Windows Package Manager version 0.1.41821 Preview
Windows: Windows.Desktop version 10.0.20161.1000
Package: Microsoft.DesktopAppInstaller version 1.0.41821.0

Any other software? Windows Subsystem for Linux, Windows Terminal, WinGet, etc.
```

**Comment by adandreou** [comment]
If you run Command Prompt or PowerShell as administrator, the result is the same?

**Comment by nitishvijai** [comment]
Nope, it successfully installed with administrator privileges. I do think it would be more convenient if it worked without administrator privileges.

**Comment by adandreou** [comment]
In my case when I tried to install it without administrator privileges the UAC prompt popped up. There is an open issue in this topic on winget-cli repo https://github.com/microsoft/winget-cli/issues/218
